beacon_node/eth1/src/deposit_cache.rs

```diff
@@ -338,87 +509,99 @@ pub mod tests {
+
+        deposit_cache
+    }
+
     #[test]
     fn insert_log_valid() {
-        let mut tree = DepositCache::default();
+        let mut deposit_cache = DepositCache::default();
 
         for i in 0..16 {
             let mut log = example_log();
             log.index = i;
-            tree.insert_log(log).expect("should add consecutive logs");
+            deposit_cache
+                .insert_log(log)
+                .expect("should add consecutive logs");
         }
     }
 
     #[test]
     fn insert_log_invalid() {
-        let mut tree = DepositCache::default();
+        let mut deposit_cache = DepositCache::default();
 
         for i in 0..4 {
             let mut log = example_log();
             log.index = i;
-            tree.insert_log(log).expect("should add consecutive logs");
+            deposit_cache
+                .insert_log(log)
+                .expect("should add consecutive logs");
         }
 
         // Add duplicate, when given is the same as the one known.
         let mut log = example_log();
         log.index = 3;
         assert_eq!(
-            tree.insert_log(log).unwrap(),
+            deposit_cache.insert_log(log).unwrap(),
             DepositCacheInsertOutcome::Duplicate
         );
 
         // Add duplicate, when given is different to the one known.
         let mut log = example_log();
         log.index = 3;
         log.block_number = 99;
-        assert!(tree.insert_log(log).is_err());
+        assert!(deposit_cache.insert_log(log).is_err());
 
         //  Skip inserting a log.
         let mut log = example_log();
         log.index = 5;
-        assert!(tree.insert_log(log).is_err());
+        assert!(deposit_cache.insert_log(log).is_err());
     }
 
     #[test]
     fn get_deposit_valid() {
         let n = 1_024;
-        let mut tree = DepositCache::default();
-
-        for i in 0..n {
-            let mut log = example_log();
-            log.index = i;
-            log.block_number = i;
-            log.deposit_data.withdrawal_credentials = Hash256::from_low_u64_be(i);
-            tree.insert_log(log).expect("should add consecutive logs");
-        }
+        let deposit_cache = get_cache_with_deposits(n);
 
         // Get 0 deposits, with max deposit count.
-        let (_, deposits) = tree
-            .get_deposits(0, 0, n, TREE_DEPTH)
+        let (_, deposits) = deposit_cache
+            .get_deposits(0, 0, n)
             .expect("should get the full tree");
         assert_eq!(deposits.len(), 0, "should return no deposits");
 
         // Get 0 deposits, with 0 deposit count.
-        let (_, deposits) = tree
-            .get_deposits(0, 0, 0, TREE_DEPTH)
+        let (_, deposits) = deposit_cache
+            .get_deposits(0, 0, 0)
             .expect("should get the full tree");
         assert_eq!(deposits.len(), 0, "should return no deposits");
 
         // Get 0 deposits, with 0 deposit count, tree depth 0.
```

Comment:
This is now identical to the test case above

Reply:
ah nice one! i removed this :)